CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change log
 
+
+## 3.3.0 (2020-04-23)
```

Comment:
Ahem, why is that 3.3.0 and the version 4.3.0?

Reply:
Ahem.. Ok I'll write the usual X.X.X version and we can update it in your new PR then!